index.html

```diff
@@ -15,6 +15,21 @@
 </head>
 
 <body>
+  <script>
+  function brokenImage(link,id){
+    //alert("broken image!");
+    console.log(id+" image broken");
+    var image=document.getElementById(id+'new');
+    var old=document.getElementById(id);
+    sub="<img src='"+link+"' alt='background picture showing a view of Vienna' />";
+    console.log(sub);
```

Comment:
Please remove.

Reply:
At least the "sub" variable with its code is necessary for showing the alternative picture, which is the very purpose of this brokenImage function. 
Only "console.log(sub)" isn't necessary for the code to work.